go/vt/mysqlctl/query.go

```diff
@@ -29,6 +29,11 @@ import (
 	"vitess.io/vitess/go/vt/log"
 )
 
+const (
```

Comment:
Can you move this near the `redactMasterPassword` function where the original consts were defined?